assembl/static2/js/app/client.js

```diff
@@ -0,0 +1,7 @@
+import ApolloClient, { createNetworkInterface } from 'apollo-client';
+
+const client = new ApolloClient({
+  networkInterface: createNetworkInterface({ uri: 'http://localhost:3000/graphql' })
```

Comment:
I think there is a backend task to do here, we should have in the page the url of the graphql endpoint, which is the domain/slug/graphql.